### DISCUSSION TITLE
Use acebase on Deno

### DISCUSSION THREAD

**Discussion by garder500** [appy-one/acebase#126] [Q&A]
Hey, can i use Acebase with Deno ?

**Comment by appy-one** [comment]
I have no experience with Deno (yet) so if anyone else does, please do share.

**Reply by garder500** [reply]
Basically, the language is the same as Nodejs, (Javascript / typescript), compared to node no need to compile the Typescript, and the import of modules is slightly different (it's done with the url) Where it's complicated this is going to be the dependencies you are currently using i think
